pkg/util/wait/wait_test.go

```diff
@@ -12,13 +12,16 @@ import (
 	"testing"
 	"time"
 
+	"github.com/sirupsen/logrus"
+
 	"github.com/golang/mock/gomock"
 
 	"github.com/openshift/openshift-azure/pkg/util/mocks/mock_wait"
 )
 
 func TestForHTTPStatusOk(t *testing.T) {
 	urltocheck := "http://localhost:12345/nowhere"
+	log := &logrus.Entry{}
```

Comment:
this will panic if used - you should use the proper recipe

Reply:
Updated.